packages/server/modules/auth/strategies/azureAd.ts

```diff
@@ -128,50 +129,29 @@ const azureAdStrategyBuilder: AuthStrategyBuilder = async (
           return next()
         }
 
-        // if the server is not invite only, go ahead and log the user in.
```

Comment:
@gjedlicska It seems that there was a long-standing bug in SSO registration - if the server isn't inviteOnly, then the active invite wasn't used at all, the user was registered without it.

I fixed this in all SSO strategies, but let me know if you think that the way it was before was actually intentional

Reply:
Lets test this on latest